packages/at_utils/lib/src/logging/atsignlogger.dart

```diff
@@ -13,19 +13,38 @@ class AtSignLogger {
 
   static final ConsoleLoggingHandler _consoleLoggingHandler =
       ConsoleLoggingHandler();
-  static final StdErrLoggingHandler _stdErrLoggingHandler =
-      StdErrLoggingHandler();
+
```

Comment:
Make _consoleLoggingHandler public, as it is final anyway
Undelete _stdErrLoggingHandler (but make it public also)

Then, while it's starting up, application code can do `AtSignLogger.defaultLoggingHandler = AtSignLogger.stdErrLoggingHandler;`